src/components/team/DomainDesktop.tsx

```diff
@@ -0,0 +1,30 @@
+import { Member } from './Member'
+import { Members } from '@components/team/MemberComponentTypes'
+
+type Props = {
+  domainName: string
+  members: {
+    nodes: Members[]
+  }
+}
+
+const DomainDesktop = ({
+  domainName,
+  members,
+}: {
+  domainName: string
+  members: any[]
```

Comment:
**Problem:** Please use a proper type.

Reply:
I feel like this feedback can be expanded upon a little, if I may.

We want to avoid using the `any` type. In order to do so, we'd need to, for example, define a "Member" type

Reply:
It would help to examine what the data entries of a Member type would look like and set these. We can base these on the data we're using lower down in the markup

Reply:
@KeShaKreates Happy to pair with you on this if you like!

Reply:
Thank you for the feedback and clarity! I have removed the any type and used the Member type as requested.